docs/contributor_guide.rst

```diff
@@ -137,7 +137,7 @@ Existing tags:
 - ``[Metal], [OpenGL], [CPU], [CUDA], [AMDGPU], [LLVM]``: backends;
 - ``[LLVM]``: the LLVM backend shared by CPUs and CUDA;
 - ``[Lang]``: frontend language features, including syntax sugars;
-- ``[Std]``: standard library, e.g. `ti.Matrix` and `ti.Vector`;
+- ``[Std]``: standard library, e.g. ``ti.Matrix`` and ``ti.Vector``;
```

Comment:
Thank for adding this, btw, could you also add a `[Benchmark]` at some point? See #981.

Reply:
Please feel free to do that on your own in a separate PR. Thanks.

Reply:
Added in #981 now!

Reply:
Thank you!